http-netty/src/test/groovy/io/micronaut/http/netty/allocator/DefaultAllocatorSpec.groovy

```diff
@@ -37,6 +38,7 @@ class DefaultAllocatorSpec extends Specification {
         'maxCachedByteBuffersPerChunk' | 20
     }
 
+    @Ignore
```

Comment:
if this reliably fails now, maybe your change caused the allocator class that reads the system property to load before the config is applied. this needs to be fixed